js/src/edit.js

```diff
@@ -132,11 +137,35 @@ class Edit {
 						} }
 					</Slot>
 				</PluginSidebar>
-
 				<Fragment>
 					<Sidebar store={ store } theme={ theme } />
 					<MetaboxPortal target="wpseo-metabox-root" store={ store } theme={ theme } />
 				</Fragment>
+				<PluginPrePublishPanel
+					className="yoast-seo-sidebar-panel"
+					title={ __( "Yoast SEO", "wordpress-seo" ) }
+					initialOpen={ true }
+					icon={ <Fragment /> }
```

Comment:
Did you intend to pass a logo here? Passing an empty `Fragment` seems odd but perhaps I don't understand the functionality.